paseos/power/is_in_eclipse.py

```diff
@@ -17,11 +17,7 @@ def is_in_eclipse(actor, central_body: pk.planet, t: pk.epoch, plot=False) -> bo
     Returns:
         bool: True if actor is in eclipse
     """
-    # on OSX this can throw an error it seems.
-    try:
-        logger.debug(f"Checking whether {actor} is in eclipse at {t}.")
-    except RuntimeError:
-        logger.debug(f"Checking whether {actor} is in eclipse at {t.mjd2000} (mjd2000).")
+    logger.debug(f"Checking whether {actor} is in eclipse at {t}.")
```

Comment:
I think this leads to crashes on OS X, please don't remove

Reply:
I guess an error due to iterated merging. Thanks, I will revert it.